src/ivc/incrementally_verifiable_computation.rs

```diff
@@ -55,7 +63,7 @@ where
     primary: StepCircuitContext<A1, C1, SC1>,
     secondary: StepCircuitContext<A2, C2, SC2>,
 
-    primary_trace: PlonkTrace<C2>,
+    secondary_trace: PlonkTrace<C2>,
```

Comment:
As for one step of IVC (i.e. RecursiveSNARK), we synthesize primary and secondary circuit, then return `3` PlonkTrace. i.e. 2 RelaxedInstance from both circuits and 1 strict from either primary or secondary. IIRC, if we start from primary, then we can skip the strict primary instance.
Here only includes one of `PlonkTrace` (RelaxedInstance of secondary circuit).